pyproject.toml

```diff
@@ -43,6 +43,6 @@ skip-string-normalization = true
 
 [tool.isort]
 include_trailing_comma = "True"
-known_third_party = ["PIL", "black", "fairlearn", "git", "matplotlib", "mypy", "numpy", "pandas", "pylint", "pytest", "scipy", "seaborn", "setuptools", "sklearn", "tap", "teext", "torch", "torchvision", "tqdm", "typing_extensions"]
+known_third_party = ["PIL", "black", "git", "matplotlib", "mypy", "numpy", "pandas", "pylint", "pytest", "scipy", "seaborn", "setuptools", "sklearn", "tap", "teext", "torch", "torchvision", "tqdm", "typing_extensions"]
```

Comment:
hm, why did you remove fairlearn?

Reply:
errrr... .I didn't, autoflake does this - It must be because fairlearn isn't is a file-top import

Reply:
I take it back, it's https://github.com/asottile/seed-isort-config and apparently it's out of date anyway. I'll remove it from the pre-commit